pypesto/objective/amici_objective.py

```diff
@@ -150,6 +142,23 @@ def __init__(self,
                     self.amici_model.getParameterScale(), len(self.edatas))
         self.mapping_scale_opt_to_scale_sim = mapping_scale_opt_to_scale_sim
 
+        # preallocate guesses, construct a dict for every edata for which we
+        # need to do preequilibration
+        if guess_steadystate:
```

Comment:
Could you add ``self.guess_steadystate = guess_steadystate`` here? I will need to update the PetabAmiciObjective later, and that will be needed there.